FSharp.AutoComplete/Program.fs

```diff
@@ -617,6 +641,36 @@ module internal Main =
 
               main state
 
+          | SymbolUse ->
+              let symboluses =
+                  async {
+                      let! symboluse = tyRes.GetSymbol(line, col, lineStr)
+                      if symboluse.IsNone then return None else
+                      let! symboluses = tyRes.GetUsesOfSymbolInFile symboluse.Value.Symbol
+                      return Some {
+                        Name = symboluse.Value.Symbol.DisplayName
+                        Uses =
+                          [ for su in symboluses do
+                              yield { StartLine = su.RangeAlternate.StartLine
+                                      StartColumn = su.RangeAlternate.StartColumn + 1
+                                      EndLine = su.RangeAlternate.EndLine
+                                      EndColumn = su.RangeAlternate.EndColumn + 1
```

Comment:
Sorry, I misread. It seems that FCS [accepts 1-based column numbers](https://github.com/fsharp/FSharp.Compiler.Service/blob/251c00058cb4ec48e9fc31a7a814d8a3f4898d3f/src/fsharp/vs/service.fsi#L189) but [returns 0-based](https://github.com/fsharp/FsAutoComplete/blob/master/FSharp.AutoComplete/test/integration/FindDeclarations/output.txt#L18). Why would this possibly be a good idea?

I've looked at the output we give and I think only `declarations`, `finddecl` and this new command return positions. I think they should all return 0-based or all return 1-based columns, for consistency. However, I think that if FCS is outputting 0-based columns it should probably be accepting 0-based as well.